### PR TITLE
Dict: PyObject can be used as keys

### PR DESCRIPTION
**Pull request by bbtfr**
I'm trying to write some tensorflow scripts with `pycall` in Ruby now.
I found that it's not possible to use PyObject as python dict key now, so I have to use, for example: `{ x.__pyobj__ => value }` instead of simply `{ x => value }`.

Here's an use case:
https://gist.github.com/bbtfr/5b3e2d20c0d8e7b0d0b716050d7a8217#file-mnist-softmax-rb-L41

### CODE REVIEW

**Comment by bbtfr** [comment]
BTW, `pycall` is an amazing idea, thanks for the gem.

**Comment by bbtfr** [comment]
no test provide and duplicate with #14 , closing it.
